docs/user/supported_devices.rst

```diff
@@ -102,6 +102,7 @@ ath79-generic
 
 * Ubiquiti
 
+  - Loco M2 XW
```

Comment:
the device should be called like the others are:
NanoStation Loco M2 (XW)

this also applies to the commit text

Reply:
Indeed, thanks

Reply:
resolved in 2d481765ee1732ec4f8066bad284800fd642b630